Source/ThirdParty/libwebrtc/Source/third_party/libyuv/BUILD.gn

```diff
@@ -217,23 +236,41 @@ if (libyuv_use_neon) {
 
     public_configs = [ ":libyuv_config" ]
 
-    # Always enable optimization for Release and NaCl builds (to workaround
-    # crbug.com/538243).
-    if (!is_debug) {
-      configs -= [ "//build/config/compiler:default_optimization" ]
-
-      # Enable optimize for speed (-O2) over size (-Os).
-      # TODO(fbarchard): Consider optimize_speed which is O3.
-      configs += [ "//build/config/compiler:optimize_max" ]
-    }
-
-    if (current_cpu != "arm64") {
+    if (current_cpu == "arm64") {
+      cflags = [ "-march=armv8-a+dotprod+i8mm" ]
```

Comment:
is this applicable for Apple Silicon ?

Reply:
though we're aarch64 ; but for macOS it shows as arm64 so it's puzzling

Reply:
We compile out i8mm and dot prod specific routines from libyuv, so I don't think this matters.
@emw-apple, thoughts?

Reply:
I don't think it matters either — absent a good reason, we should stick to the default compiler settings provided by our SDKs.